### PR TITLE
DOCS-7747 Add DORA Metrics Data Collected Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Removes duplicate information in the product landing page and the Setup page, adds a Data Collected page, and screenshots of the DORA Metrics page.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by rtrieu** [comment]
Created DOCS-7751 for editorial review.

File: content/en/dora_metrics/data_collected/_index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /service_management/events/explorer/
+[2]: /api/latest/metrics/#query-timeseries-points
+[3]: /api/latest/metrics/#query-timeseries-data-across-multiple-products
+[4]: /service_management/events/
+[5]: https://app.datadoghq.com/event/explorer?query=source%3Asoftware_delivery_insights
+[6]: /getting_started/tagging/unified_service_tagging/
```
Comment (aliciascott) [inline content/en/dora_metrics/data_collected/_index.md:55]:
```suggestion
[6]: /getting_started/tagging/unified_service_tagging/
[7]: /api/latest/dora-metrics/
```
would be good to link to the API here

File: content/en/dora_metrics/_index.md [hunk 2]
```diff
@@ -47,65 +44,35 @@ Change Failure Rate
...
 [1]: https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance
-[2]: https://app.datadoghq.com/ci/dora
-[3]: https://docs.datadoghq.com/monitors/types/metric/?tab=threshold
-[4]: https://docs.datadoghq.com/api/latest/metrics/#query-timeseries-points
-[5]: https://docs.datadoghq.com/api/latest/metrics/#query-timeseries-data-across-multiple-products
-[6]: https://app.datadoghq.com/event/explorer?query=source%3Asoftware_delivery_insights
-[7]: https://docs.datadoghq.com/service_management/events/explorer/
+[2]: /dora_metrics/setup/
+[3]: /dora_metrics/data_collected/
+[4]: https://app.datadoghq.com/ci/dora
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/dora_metrics/_index.md:76]: Are you in the Demo instance? We should see the Getting Started blank page:

Screenshot 2024-04-24 at 4 20 37 PM

File: content/en/dora_metrics/_index.md [hunk 3]
```diff
@@ -47,65 +44,35 @@ Change Failure Rate
...
+[5]: /monitors/types/metric/?tab=threshold
+[6]: /monitors/
```
Comment (alai97) [inline content/en/dora_metrics/_index.md:78]:
```suggestion
[6]: /monitors/
[7]: https://app.datadoghq.com/event/explorer
```

File: content/en/dora_metrics/_index.md [hunk 4]
```diff
@@ -47,65 +44,35 @@ Change Failure Rate
...
+{{< img src="dora_metrics/home.png" alt="An Overview of your DORA Metrics calculations and insights in the last week" style="width:100%;" >}}
 
-### Deployment and incident events
+You can access and visualize your DORA metrics and filter them by team, service, repository, environment, and time period on the [**DORA Metrics** page][4]. For more information about the metrics calculated by DORA Metrics, see the [Data Collected documentation][3].
 
-DORA Metrics also provides individual `deployment`, `incident`, and `incident_finished` events in [Event Management][6] with `source:software_delivery_insights`.
+## Examine DORA Metrics events
 
-The events can be queried and visualized with the [Events Explorer][7].
+DORA Metrics provides individual `deployment`, `incident`, and `incident_finished` events under the `Software Delivery Insights` source in [Event Management][4]. Navigate to [**Service Management** > **Event Management** > **Explorer**][5] and enter `source:software_delivery_insights` in the search query to filter on DORA Metrics events.
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/dora_metrics/_index.md:63]: Updated, thanks for flagging!

File: content/en/dora_metrics/data_collected/_index.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+All default metrics contain the following tags if any are available:
+
+- `service`
+- `team`
+- `env`
+- `repository_id`
+
+**Note**: The `severity` tag is available for `dora.incidents_impact` and `dora.time_to_restore` metrics, if it is provided through the [DORA Metrics API][7].
+
+For more information about using reserved `env`, `service`, and `version` tags, see [Unified Service Tagging][6].
```
Comment (alai97) [inline content/en/dora_metrics/data_collected/_index.md:44]:
```suggestion
For more information about using `env`, `service`, and `version` tags, see [Getting Started with Tags][6].
```

File: content/en/dora_metrics/data_collected/_index.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /service_management/events/explorer/
+[2]: /api/latest/metrics/#query-timeseries-points
+[3]: /api/latest/metrics/#query-timeseries-data-across-multiple-products
+[4]: /service_management/events/
+[5]: https://app.datadoghq.com/event/explorer?query=source%3Asoftware_delivery_insights
+[6]: /getting_started/tagging/unified_service_tagging/
```
Comment (alai97) [inline content/en/dora_metrics/data_collected/_index.md:55]:
```suggestion
[6]: /getting_started/tagging/
```

**Pull request closed without merging**